lib/routes/gov/news/index.js

```diff
@@ -35,6 +35,8 @@ module.exports = async (ctx) => {
     const $ = cheerio.load(listData.data);
     if (url.search(/zhengcelibrary/g) !== -1) {
         list = $('.dys_middle_result_content_item');
+    } else if (url.search(/bumen/g) !== -1) {
```

Comment:
ae480b8b49d86188cc6aee796c7e817767143bd7 uses `.includes()` instead of a double negative `.search() !== -1`